test.rb

```diff
@@ -17,12 +17,12 @@ def interpolation
   end
 
   def condition
-    if true then 1 else 2 end
     true ? 1 : 2
+    if true then 1 else 2 end
```

Comment:
Favor the ternary operator (`?:`) over `if/then/else/end` constructs.<br>Literal `true` appeared in a condition.